bundles/org.openhab.binding.androiddebugbridge/README.md

```diff
@@ -10,7 +12,7 @@ This binding was tested on the Fire TV Stick (android version 7.1.2, volume cont
 
 ## Discovery
 
-As I can not find a way to identify android devices in the network the discovery will try to connect through adb to all the reachable ip in the defined range, you could customize the discovery process through the binding options. **Your device will prop a message requesting you to authorize the connection, you should check the option "Always allow connections from this device" (or something similar) and accept**.
+As I can not find a way to identify android devices in the network the discovery will try to connect through adb to all the reachable ip in the defined range, you could customize the discovery process through the binding options. **Your device will prompt a message requesting you to authorize the connection, you should check the option "Always allow connections from this device" (or something similar) and accept**.
```

Comment:
I believe you should end your sentence after "range". And please put each sentence on a different line.